ykrt/src/compile/jitc_yk/jit_ir.rs

```diff
@@ -92,6 +99,18 @@ macro_rules! index_16bit {
                 self.0.into()
             }
         }
+
+        // impl From<usize> for $struct {
```

Comment:
Left over comments?

Reply:
ug, yes.

Reply:
Fixed in https://github.com/ykjit/yk/pull/971/commits/3eb9bac52fff8d291f2540a76e2644b5520bbe76